### PR TITLE
Hazim/feature/right sidebar

### PR DESCRIPTION
**Pull request by ZimLim**
# Description

> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Added right sidebar with filters and description
- Ability to add new filters from sidebar
- Added autocomplete
## Checklist

- [x] This PR can be reviewed in under 30 minutes
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned reviewers to this PR.

### CODE REVIEW

File: src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -156,97 +196,125 @@ export const DashboardPage: FC = () => {
...
+                    maxHeight: '20vh'
+                  }
+                }}
+                onInputChange={(_event, newInputValue) => {
+                  setInputValue(newInputValue);
+                }}
+                inputValue={inputValue}
+                value={selectedCategories}
+                onChange={handleAddFilterAutocomplete}
               />
```
Comment (wenhwang97) [inline src/pages/Dashboard.tsx]: Something went wrong here. `<Autocomplete>` is not doing its job. When you are typing in the textfield, the dropdown never changes. I did a quick investigation and the issue happened here #21 .

**Review by wenhwang97 — COMMENTED** [review]
I highlighted the `<Autocomplete>` issue. Surprised that it wasn't discovered at that time. Probably need to create a separate PR just to fix this issue.

File: src/pages/Dashboard.tsx [hunk 2]
```diff
@@ -86,6 +111,21 @@ export const DashboardPage: FC = () => {
     });
   };
 
+  const handleAddFilter = (newValue: string) => {
+    console.log(newValue)
```
Comment (wenhwang97) [inline src/pages/Dashboard.tsx]: Remove this printing statement

**Review by wenhwang97 — CHANGES_REQUESTED** [review]
Look very nice! Just some nitpicks. Make sure to run prettier to clean up the code formatting.

File: src/pages/Dashboard.tsx (hunk 1 above)
Comment (ZimLim) [reply src/pages/Dashboard.tsx]: I didn't particularly pay attention to it but I can make the fix separately

**Review by wenhwang97 — CHANGES_REQUESTED** [review]
Need to run `npm run prettier:fix` to fix the formatting

**Review by wenhwang97 — APPROVED** [review]

**Pull request merged**